ui-lib/src/components/Divider/index.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+type DividerTypes = "light" | "dark";
+
+interface Props {
+  margin?: number;
```

Comment:
*I would like to add one more thing, I know that was the scope of the ticket, that's why I separate this comment from the PR review.*

---

For me, components must not use margin propriety, this is not their role. We should imagine components like a capsule and only define **internal style**.
That isn't the role of atoms, molecules or organisms to have an impact on the layout of a template or the pages. 

![image](https://user-images.githubusercontent.com/33158502/129708011-9d774efe-b684-473a-967a-893ba314faf0.png)


By setting the margin here, we go beyond the design definition because designers don't define external spaces on Figma. In fact, they will use grid/flexbox systems to apply some spacing rules, depending on the context (e.g define a larger spacing when the Divider is between two sections than between two paragraphs) and that's exactly what we will. When we'll start to integrate templates and pages we'll use Flexbox and Grid components to define layouts and use gaps to separate components in a harmony way. By allowing the possibility to set margin here, we mix atoms/molecules roles with layout/pages roles, the components receive two specific responsibilities which is one too many.

I am deeply convinced that is not a good way to go and I think that we should adopt the atomic [design methodology created by Brad Frost](https://bradfrost.com/blog/post/atomic-web-design/). (the book is a must-read btw). I will be happy to discuss with the team about that.

Reply:
I trust you on this, up to you to convince the rest of the team though